### ISSUE TITLE
IcingaCheckDirectory:  FileSizeSmallerThan behaves like FileSizeGreaterThan

### ISSUE THREAD

**Issue by mrdsam** [Icinga/icinga-powershell-plugins#166]
## Problem
`Invoke-IcingaCheckDirectory -FileSizeSmallerThan` has the same effect as `Invoke-IcingaCheckDirectory -FileSizeGreaterThan`

## Expected behavior
Option  -FileSizeSmallerThan should find files smaller than the given value, not files which are geater.

## Version
v1.4

## Solution
The culprit seems to be in "Icinga_Provider_Directory.psm1" in line 117. But I'm not familiar with powershell, so don't be angry that I don't send a PR.

```powershell
117       $DirectoryData = ($DirectoryData | Where-Object {$_.Length -gt $FileSizeSmallerThanValue})
```
I think it should be
```powershell
117       $DirectoryData = ($DirectoryData | Where-Object {$_.Length -lt $FileSizeSmallerThanValue})
```

**Comment by LordHepipud** [comment]
Hello,

thank you for the issue. This should already be fixed in the current master for 1.5.0: #147 
Can you confirm this?

**Comment by mrdsam** [comment]
I see, thanks!
